ambari-server/src/main/java/org/apache/ambari/server/state/stack/UpgradePack.java

```diff
@@ -262,12 +291,6 @@ public void mergeProcessing(UpgradePack pack) {
     return intermediateStacks;
   }
 
-  /**
-   * @return the target stack, or {@code null} if the upgrade is within the same stack
```

Comment:
is it true that the target stack can not be Null now? This part of the javadoc is missing in a new getter

Reply:
Will update next patch.